### DISCUSSION TITLE
add pages E-COMMERCE (product overview, shopping cart, .. ) and APPS (Blog, ..) into Poseidon template

### DISCUSSION THREAD

**Discussion by stcsdmm7** [primefaces/community#974] [PrimeNG] [answered]
Hi,

Is there any chance to add pages E-COMMERCE (product overview, shopping cart, .. ) and APPS (Blog, ..) into Poseidon template

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

It's currently in progress. We've added all of them into the Poseidon, it's in the review process and the updated version will be released in upcoming weeks.
